src/main/java/org/spongepowered/common/service/pagination/PaginationCalculator.java

```diff
@@ -85,37 +89,89 @@ public int getLinesPerPage(MessageReceiver source) {
         return this.linesPerPage;
     }
 
-    public int getLines(MessageReceiver source, Text text) {
-        return (int) Math.ceil((double) getLength(source, text) / LINE_WIDTH);
+    /**
+     *
+     * @param text
+     * @return
+     */
+    public int getLines(Text text) {
+        return (int) Math.ceil((double) getLength(text) / LINE_WIDTH);
     }
 
-    private double getWidth(int codePoint, boolean isBold) {
+    private static class thing{
+        private final char i;
```

Comment:
What do these 1-character fields represent? Giving them real names would be nice.